src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2192,6 +2196,21 @@ def test_vmss_create_x_zones(self, resource_group, resource_group_location):
             self.check('[0].zones', None)
         ])
 
+        # Now provision a web server
+        self.cmd('network lb probe create -g {rg} --lb-name {lb} -n {probe} --protocol http --port 80 --path /')
+        self.cmd('network lb rule create -g {rg} --lb-name {lb} -n {rule} --protocol tcp --frontend-port 80 --backend-port 80 --probe-name {probe}')
+        self.cmd('network nsg rule create -g {rg} --nsg-name {nsg} -n allowhttp --priority 4096 --destination-port-ranges 80 --protocol Tcp')
+
+        self.cmd('vmss extension set -g {rg} --vmss-name {vmss} -n customScript --publisher Microsoft.Azure.Extensions --settings "{{\\"commandToExecute\\": \\"sudo apt-get install -y nginx\\"}}" --version 2.0')
+        self.cmd('vmss update-instances -g {rg} -n {vmss} --instance-ids "*"')
```

Comment:
//CC: @gatneil  @iainfoulds 
This is to consolidate tests to keep me honest that the x-zone VMSS created by CLI can be relatively easily configured to become web servers

Reply:
Yup! This looks good to me :). Glad to see an end-to-end scenario test for this.